app/views/backend/entities/_form.html.haml

```diff
@@ -65,7 +62,6 @@
   = field_set(:employee_informations) do
     = f.input :employee, show: '#employee-informations'
     #employee-informations
-      - scopes = { employees: true, not_centralizing: true}
-      = f.referenced_association :employee_account, source: {scope: scopes}, buttons: {new: { number: Account.find_or_import_from_nomenclature(:staff_due_remunerations).number}}
+      = f.referenced_association :employee_account, source: :employees, new: { number: Account.find_or_import_from_nomenclature(:staff_due_remunerations).number}
```

Comment:
See what we can do about persistence here? (`import` prbably persists the account if it isn't found)